toqito/channels/depolarizing.py

```diff
@@ -69,11 +69,12 @@ def depolarizing(dim: int, param_p: float = 0) -> np.ndarray:
     >>> test_input_mat = np.array(
     ...     [[1, 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12], [13, 14, 15, 16]]
     ... )
-    >>> apply_channel(test_input_mat, depolarizing(4, 0.5))
-    array([[ 4.75,  1.  ,  1.5 ,  2.  ],
-           [ 2.5 ,  7.25,  3.5 ,  4.  ],
-           [ 4.5 ,  5.  ,  9.75,  6.  ],
-           [ 6.5 ,  7.  ,  7.5 , 12.25]])
+    >>> apply_channel(test_input_mat, depolarizing(4, param_p=1))
```

Comment:
I don't understand why you changed `param_p = 1` for all the examples?

Reply:
never mind, I'll take the rest of the issue from here.

Reply:
But it doesn't require an argument--a default is provided:
https://github.com/vprusso/toqito/blob/1379480fc43a9123e141fbd0794eaa8851cb6ec9/toqito/channels/depolarizing.py#L8